src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Try to rewrite the code so that it is not necessary to disable the linter

Reply:
I forgot to remove it, I was testing the doubleClick in dev tools=)